baseline/client/admin/download/style.css

```diff
@@ -1,4 +1,4 @@
-#experiment-download[aria-disabled="true"] {
```

Comment:
Was this styling removed intentionally because it was unnecessary, or should it be applied to download-status?

Reply:
That rule was from back when `#experiment-download` was a link added to the download page when the link becomes ready! Now, it should be unnecessary.